scylla/tests/integration/session.rs

```diff
@@ -831,6 +831,39 @@ async fn test_get_tracing_info(session: &Session, ks: String) {
     let tracing_info: TracingInfo = session.get_tracing_info(&tracing_id).await.unwrap();
     assert!(!tracing_info.events.is_empty());
     assert!(!tracing_info.nodes().is_empty());
+
+    // Check if the request type matches
+    assert_eq!(tracing_info.request.unwrap(), "Execute CQL3 query");
+
+    // Verify duration is positive when Scylla is used
+    if session
+        .get_cluster_state()
+        .get_nodes_info()
+        .first()
+        .unwrap()
+        .sharder()
+        .is_some()
+    {
+        assert!(tracing_info.duration.unwrap() > 0);
+    }
```

Comment:
Is there some Scylla issue about the incompatibility? Or maybe a documentation if the incompatibility is intentional? It should be linked here.

Reply:
I adjusted the test to verify behavior when cassandra is used (waiting for duration to appear - as cassandra has some delay)